docs/en/02-how-to-run/useful_tools.md

```diff
@@ -203,6 +203,23 @@ And the output look like this:
 +--------+------------+---------+
 ```
 
+## visualize
+
+This tool can be used to visualize model inference results in different backend.
+
+### Usage
+
+```bash
+python tools/visualize.py \
+    --deploy-cfg {DEPLOY_CFG} \
+    --model-cfg {MODEL_CFG} \
+    --deploy-path {DEPLOY_PATH} \
+    --test-img {TEST_IMGS} \
```

Comment:
Could we support batch inference in this tool?

Reply:
ok，I will try it :)

Reply:
@RunningLeon Hello, it seems a little hard to support batch inference.

In this script, I mainly use the function `visualize_model` to get inference results and draw results. It use iteration to get each result in model, like:

```python
    for single_img in img:
        task_processor.visualize(
...
```

If use batch to get inference result, maybe need to use multiprocess to parallelize visualization. Besides, I need to concat model results with inference engine and PyTorch, so I need to serialize the inference pipeline.

So can you give me some advices in it?

Reply:
@IRONICBo hi, could we add some constraints here. For instance, batch can only be in [1, 2] and we do not directly use `visualize_model` and write your own pipeline.
for instance
```
task_processor = build_task_processor(model_cfg, deploy_cfg, device)
pytorch_model = task_processor.build_pytorch_model(ckpt_path) # if ckpt is provided
backend_model = task_processor.build_backend_model(backend_model_path)
model_inputs, _ = task_processor.create_input(img, input_shape)
for batch_imgs in imges:
    model_inputs, _ = task_processor.create_input(batch_imgs, input_shape)
    pytorch_results = pytorch_model.test_step(model_inputs)[0]
    backend_results = backend_model.test_step(model_inputs)
    task_processor.visualize()...
```